ui/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euv -o pipefail
+
+perform_push="${PERFORM_PUSH-false}"
+
+repository=shepmaster
+
+# Build outside the image so nodejs isn't installed on the main image
```

Comment:
I can't wait for [multi stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/) to land on a version of Docker we have!